package.json

```diff
@@ -4,7 +4,7 @@
 	"description": "Node JS Api",
 	"main": "./dist/server.js",
 	"scripts": {
-		"docker_build": "docker build -t goodbuy/nodejs-webserver .",
+		"docker_build_and_push": "docker build -t goodbuy/nodejs-webserver . && docker push goodbuy/nodejs-webserver",
```

Comment:
What is the reason behind the change? 
I think we first have to build the image and test it bevor we can push it. 
And if we already build it we don't have to build it again to push it. 
Maybe just leave docker_build and make an extra script docker_push?

Reply:
We actually talked about this change in person, but be my guest:

The "old" docker build command has no use anymore since we use to tag images tagged with our names. So we talked about to change the docker_build command, so we can use it after a branch is merged into dev. Lile this we don't forget to also push the new image to our docker repo. 

But I 'also' think it's 'dangerous' to have a script like this, caus somebody could build and push a wrong image, and then we don't know how to roll back, at least at the moment.

So I actually would say, let's remove this command completely for now and implement the build and tag step in a CI/CD.

Reply:
The CI/CD part sounds good but i like the build command i use it a lot otherwise i always have to type the command.